client/web/components/Flag/index.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import * as React from 'react';
+import { Image } from 'react-native'; // esling-disable-line
```

Comment:
I am pretty sure this eslint comment does not work. 😊 

```diff
- // esling-disable-line
+ // eslint-disable-line
```

Do you need to copy-paste it everywhere when you import `react-native` library?